mcux/hal_nxp.cmake

```diff
@@ -215,6 +215,7 @@ include_driver_ifdef(CONFIG_DMA_MCUX_SMARTDMA		smartdma	driver_lpc_smartdma)
 include_driver_ifdef(CONFIG_DAC_MCUX_LPDAC			dac_1		driver_dac_1)
```

Comment:
Commit message says CONFIG_DMA_NXP_IMX_EDMA but the actual code in hal_nxp.cmake uses CONFIG_DMA_NXP_EDMA, we need some clarifications here.

Reply:
Fixed. Forgot to change commit message after changing the naming scheme on Zephyr side.